src/FSharp.Core/QueryExtensions.fs

```diff
@@ -17,269 +17,368 @@ open System.Linq.Expressions
     /// (pattern returns only property assignments)
-    let (|PropSetList|_|) varArg (list:Expr list) =
-        let rec propSetList acc x = 
-            match x with 
+    let (|PropSetList|_|) varArg (list: Expr list) =
+        let rec propSetList acc x =
+            match x with
             // detect " v.X <- y"
-            | ((Patterns.PropertySet(Some(Patterns.Var var), _, _, _)) as p) :: xs when var = varArg ->
+            | ((Patterns.PropertySet (Some (Patterns.Var var), _, _, _)) as p) :: xs when var = varArg ->
                 propSetList (p :: acc) xs
             // skip unit values
             | (Patterns.Value (v, _)) :: xs when v = null -> propSetList acc xs
             // detect "v"
-            | [Patterns.Var var] when var = varArg -> Some acc
+            | [ Patterns.Var var ] when var = varArg -> Some acc
             | _ -> None
+
         propSetList [] list
 
     /// Recognize object construction written using 'new O(Prop1 = <e>, Prop2 = <e>, ...)'
-    let (|ObjectConstruction|_|) e = 
+    let (|ObjectConstruction|_|) e =
         match e with
-        | Patterns.Let ( var, (Patterns.NewObject(_, []) as init), LeftSequentialSeries propSets ) ->
-            match propSets with 
+        | Patterns.Let (var, (Patterns.NewObject (_, []) as init), LeftSequentialSeries propSets) ->
+            match propSets with
             | PropSetList var propSets -> Some(var, init, propSets)
             | _ -> None
         | _ -> None
 
 
 
     // Get arrays of types & map of transformations
-    let tupleTypes = 
-      [|  typedefof<System.Tuple<_>>,               typedefof<AnonymousObject<_>>
-          typedefof<_ * _>,                         typedefof<AnonymousObject<_, _>>
-          typedefof<_ * _ * _>,                     typedefof<AnonymousObject<_, _, _>>
-          typedefof<_ * _ * _ * _>,                 typedefof<AnonymousObject<_, _, _, _>>
-          typedefof<_ * _ * _ * _ * _>,             typedefof<AnonymousObject<_, _, _, _, _>>
-          typedefof<_ * _ * _ * _ * _ * _>,         typedefof<AnonymousObject<_, _, _, _, _, _>>
-          typedefof<_ * _ * _ * _ * _ * _ * _>,     typedefof<AnonymousObject<_, _, _, _, _, _, _>>
-          typedefof<_ * _ * _ * _ * _ * _ * _ * _>, typedefof<AnonymousObject<_, _, _, _, _, _, _, _>> |]
+    let tupleTypes =
+        [| typedefof<System.Tuple<_>>, typedefof<AnonymousObject<_>>
+           typedefof<_ * _>, typedefof<AnonymousObject<_, _>>
+           typedefof<_ * _ * _>, typedefof<AnonymousObject<_, _, _>>
+           typedefof<_ * _ * _ * _>, typedefof<AnonymousObject<_, _, _, _>>
+           typedefof<_ * _ * _ * _ * _>, typedefof<AnonymousObject<_, _, _, _, _>>
+           typedefof<_ * _ * _ * _ * _ * _>, typedefof<AnonymousObject<_, _, _, _, _, _>>
+           typedefof<_ * _ * _ * _ * _ * _ * _>, typedefof<AnonymousObject<_, _, _, _, _, _, _>>
+           typedefof<_ * _ * _ * _ * _ * _ * _ * _>, typedefof<AnonymousObject<_, _, _, _, _, _, _, _>> |]
```

Comment:
Yes, because the alignment represented a [dreaded vanity alignment](https://www.youtube.com/watch?v=cphNpqKpKc4&ab_channel=QualityReactions).
In all seriousness, the advantage of not having them aligned is that don't need to update all the lines if a longer entry is added.

Reply:
Agreed, this is ok.  There are a couple of places where this is highly problematic (the flags in TypedTree.fs for example) and we might not use auto-formatting there